Makefile

```diff
@@ -1,10 +1,14 @@
+SCHEMAS_VERSION=`cat .schemas-version`
```

Comment:
`load-schemas` needs to be chained with the `build` command. So `make build` should load DS templates, load schemas, handle translations.